src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -124,7 +128,7 @@ function bindCopyButton() {
                 $modalSessionName.val(firstSessionName);
```

Comment:
This should be here instead: `updateCharLeft('modalCopiedSessionName');`

Also, let's move `$('#copyModal').modal('show');` down after `updateCharLeft(...)`.